.github/csc.json

```diff
@@ -4,7 +4,7 @@
             "owner": "csc",
             "pattern": [
                 {
-                    "regexp": "^([^\\s].*)\\((\\d+)(?:,\\d+|,\\d+,\\d+)?\\):\\s+(error|warning)\\s+([a-zA-Z]+(?<!MSB)\\d+):\\s*(.*?)\\s+\\[(.*?)\\]$",
+                    "regexp": "^([^\\s].*)\\((\\d+)(?:,\\d+|,\\d+,\\d+)?\\):\\s+(error|warning)\\s+([a-zA-Z]+(?<!MSB)\\d+):\\s*(.*?)\\s+\\[([^\\s]+)[\\s\\W\\dA-z].*\\]$",
```

Comment:
I'm not sure we can assume that the path to the project file doesn't contain whitepace--it's generally legal for it to do so. Should we separate based on a different character, or can this be rewritten to work backwards from the end to a space instead?

Reply:
A file path can't contain < so I could use that character before the properties are printed out to separate the file path and the properties

Reply:
I changed it so the output should follow below. I think this actually makes it a bit more clearer as well as to the separation between the properties and the file path S:\msbuild\src\Build\Evaluation\ExpressionShredder.cs(33,7): error CS1003: Syntax error, ',' expected [S:\msbuild\src\Build\Microsoft.Build.csproj > Properties:prop]

Reply:
The new pattern leaves the space between the file name and the greater-than-sign which might mess with the problem matching

Reply:
Right. I'll take that space out of the new pattern

Reply:
I hope no one ever does this, but < and almost any other character can be used in filenames on linux.